JS-Week-3/TimeCalculation.js

```diff
@@ -0,0 +1,14 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  }
+function timeCalculation(travelInformation){
+    let totalTravelTime =   travelInformation.destinationDistance/travelInformation.speed;
+    const hours = Math.floor(totalTravelTime);
+const minutes = Math.round((totalTravelTime - hours) * 60);
```

Comment:
Please, remember to format with `Prettier` so indentation and spacing is consistent everywhere. You can also set it up so it does it automatically each time you save a file. Follow this guide - https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code

Reply:
H @jason-vasilev i have done the changes , please check the code.